src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -76,49 +79,26 @@ const formConfig = {
   formId: '21-526EZ',
   version: 1,
   migrations: [],
+  prefillTransformer,
   savedFormMessages: {
     notFound: 'Please start over to apply for disability claims increase.',
     noAuth: 'Please sign in again to resume your application for disability claims increase.'
   },
+  storePrefillStatus: true,
```

Comment:
Is this necessary now?

Reply:
That is zombie code now, thanks for catching it. Removed.